includes/abstracts/abstract-wc-order.php

```diff
@@ -174,6 +174,11 @@ public function save() {
 			}
 		}
 		$this->save_items();
+
+		if ( $this->data_store ) {
+			do_action( 'woocommerce_after_' . $this->object_type . '_object_save', $this, $this->data_store );
```

Comment:
This should be added across all CRUD classes if included.

Reply:
👍 makes sense. Fixed in 9f93cd6e7.